doc/whatsnew/fragments/7281.new_check

```diff
@@ -0,0 +1,4 @@
+Add ``magic-number`` plug-in checker for comparison with constants instead of named constants or enums.
```

Comment:
Name should be updated here :smile: I think `plug-in` can be written as `plugin`.